### PR TITLE
Add documentation for the `require` env processor

### PR DESCRIPTION
**Pull request by mpdude**
Documentation for https://github.com/symfony/symfony/pull/30897, discussed on the FOSSHackathon.

### CODE REVIEW

**Review by maxhelias — APPROVED** [review]

File: configuration/environment_variables.rst [hunk 1]
```diff
@@ -445,6 +445,47 @@ Symfony provides the following env var processors:
                 'auth' => '%env(file:AUTH_FILE)%',
             ]);
 
+``env(require:FOO)``
+    ``require()`` the PHP file whose path is the value of the ``FOO``
+    env var and return the value ``return``ed from it.
```
Comment (wouterj) [inline configuration/environment_variables.rst]: This is not valid reStructuredText. Let's just do `returned` as a word

**Review by wouterj — COMMENTED** [review]
Thanks for creating the documentation directly after submitting the PHP code!!

Once it's merged in the code, we should not forget to add a versionadded directive indicating when this feature was introduced.

File: configuration/environment_variables.rst (hunk 1 above)
Comment (mpdude) [reply configuration/environment_variables.rst]: fixed

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please add a versionadded directive like for the url env car processor

**Comment by mpdude** [comment]
@OskarStark latest version added a versionadded notice

**Review by OskarStark — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Review by dbrumann — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by wouterj** [comment]
Thanks again Matthias!

**Pull request merged**